lib/base.js

```diff
@@ -329,8 +258,32 @@ BaseElement = createClass({
     }
   },
 
-  // TODO: Mixin these mutator functions to share code with Meta class
-  // Subclassing causes circular redundancy
+  meta: {
+    get: function() {
+      if (!this._meta) {
+        this._meta = registry.toElement({});
```

Comment:
`#toElement` takes a primitive value and converts it to an element. You can give it number, `true`, etc.

```js
registry.toElement(0);
```